src/main/java/uk/gov/hmcts/sptribs/ciccase/CriminalInjuriesCompensation.java

```diff
@@ -28,6 +29,7 @@ public void configure(final ConfigBuilder<CaseData, State, UserRole> configBuild
         configBuilder.jurisdiction(JURISDICTION, "CIC", CcdServiceCode.CIC.getCcdServiceDescription());
 
 
+
```

Comment:
Both of these seem a bit redundant

Reply:
These 2 lines of code has to be there until we resolved all the issues related to our Jurisdiction (ST_CIC). Once we get everything working then we will remove these 2 lines. that is what i think.